themes/default/content/docs/get-started/install/_index.md

```diff
@@ -15,20 +15,47 @@ NOTE: To update this page with a new binary release, do the following:
 - Update `content/docs/get-started/install/versions.md`
 -->
 
-This page contains detailed instructions for [installing Pulumi](#installing-pulumi) on your machine. For links to detailed release notes, see the [Available Versions]({{< relref "versions" >}}) page.
+{{< chooser os "macos,windows,linux" >}}
 
-{{< get-started-note >}}
+{{% choosable os macos %}}
 
-## Installing Pulumi
+<div class="md:flex flex-row mt-6 mb-6">
+<div class="w-1/2 border-solid border-b-2 border-gray-200">
+<h3 class="no-anchor pt-4"><i class="fas fa-box pr-2"></i>Package Manager</h3>
 
-{{< chooser os "macos,windows,linux" >}}
+```bash
+$ brew install pulumi/tap/pulumi
+```
 
-{{% choosable os macos %}}
+</div>
+<div class="w-1/2 border-solid ml-4 border-b-2 border-gray-200">
+<h3 class="no-anchor pt-4"><i class="fas fa-download pr-2"></i>macOS Binary Download</h3>
+<p class="text-center">
+<a href="https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-darwin-x64.tar.gz">amd64</a>
+<a href="https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-darwin-arm64.tar.gz">arm64</a></p>
+</div>
+</div>
 
 macOS Sierra (10.12) or later is required.
 
-### Official Pulumi Homebrew Tap
+The latest version of Pulumi is {{< latest-version >}}. For older versions, see [Available Versions]({{< relref "/docs/get-started/install/versions" >}}).
+
+{{< get-started-note >}}
+
+## Other Installation Methods
+
+In addition to the above, there are many ways to install Pulumi described below:
```

Comment:
```suggestion
In addition to the above, there are many ways to install Pulumi described:
```

directional words are rarely accessible, its better to link to the section you are describing or drop them entirely